src/components/WwwFrame/WwwPage.vue

```diff
@@ -84,15 +88,31 @@ export default {
 		}
 
 		this.isKivaAppReferral = this.$route?.query?.kivaAppReferral === 'true';
+
+		const tyBadgeExpData = this.apollo.readFragment({
+			id: `Experiment:${THANKS_BADGES_EXP}`,
+			fragment: experimentVersionFragment,
+		}) ?? {};
+
+		if (tyBadgeExpData?.version) {
+			this.enableTYBadgesExp = tyBadgeExpData?.version === 'b';
+		}
 	},
 	computed: {
 		mainClasses() {
 			return [
 				this.mainClass,
 				{ 'tw-bg-secondary': this.grayBackground },
+				{ 'tw-relative': this.badgesExperimentEnabled }
```

Comment:
Can this be applied to the `WwwPage` from the thanks page?

Reply:
Sounds good. I was just thinking there might be a way to consolidate the experiment logic to the thanks page.